Components/ProductToShop.php

```diff
@@ -279,6 +283,9 @@ public function insertOrUpdate(Product $product)
             $model->setSupplier($supplier);
         }
 
+        //set product properties
+        $this->applyProductProperties($model, $product);
```

Comment:
There isn't test to confirm that product properties are really imported in shopware article.